Code/Tools/FBuild/FBuildCore/WorkerPool/JobQueue.cpp

```diff
@@ -98,26 +98,28 @@ Job * JobSubQueue::RemoveJob()
         return nullptr;
     }
 
```

Comment:
If this assert ever fires, there is a serious problem somewhere else in the code. We should not silently try to cope with this.